content/articles/geospatial_standards/index.Rmd

```diff
@@ -0,0 +1,92 @@
+---
+title: "Meet some popular open geospatial standards!"
+description: "A short introduction to the powerful GeoPackage, GeoJSON and GeoTIFF standards"
+author: "Floris Vanderhaeghe"
+date: 2019-11-19
+csl: ../inbo.csl
+bibliography: ../reproducible_research.bib
+categories: ["r", "gis"]
+tags: ["gis", "r", "open science"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+Some inspiration for this post came from the beautiful books of @lovelace_geocomputation_2019, @pebesma_edzer_spatial_2019 and @heijmans_spatial_2019, and from various websites.
+
+## Why use open standards?
+
+- Open file standards ease collaboration, portability and compatibility between users, machines and applications.
+- Their (file) structure is fully documented.
+  - Consequently, scientists and programmers can build new software / packages and make innovations that use these standards, while maintaining interoperability with existing applications.
+  - And, it guarantees that your data will still be readable in a hundred years from now, independently of which IT corporation or software is dominant at that time...
+  
+Luckily, quite a list of open standards is available!
+Below, some powerful and widely-used single-file formats are introduced.
+Single-file data sources are readily amenable to exchange and publication.
+
+I see you can't wait to start practicing, so you can also head straight over to the [tutorial on vector formats](../../tutorials/spatial_standards_vector/) and the [tutorial on the GeoTIFF raster format](../../tutorials/spatial_standards_raster/)!
+
+## A few words on the GDAL library
+
+**[GDAL](https://gdal.org)** (Geospatial Data Abstraction Library) is by far the most used collection of open-source drivers for:
+
+- [a lot](https://gdal.org/drivers/raster/index.html) of raster formats;
+- [a lot](https://gdal.org/drivers/vector/index.html) of vector formats.
+
+In other words, it is the preferred workhorse for reading and writing many geospatial file formats, used in the background by [a lot](https://gdal.org/software_using_gdal.html#software-using-gdal) of geospatial applications.
+Using GDAL is the easiest way to conform to open standards.
+
+So, in R we use packages that use GDAL in the background, such as `rgdal`, `sp`, `sf`, `raster` and `stars`.
+
+## The GeoPackage file format
```

Comment:
Sounds good to me. However, at an ArcGis [site](https://desktop.arcgis.com/en/arcmap/10.3/manage-data/administer-file-gdbs/file-geodatabases.htm) I read:

>A file geodatabase is a collection of files in a folder on disk that can store, query, and manage both spatial and nonspatial data.

So maybe not that similar, after all.

Reply:
Indeed, a *personal* geodatabase is what I've used a couple of years ago and is probably closer in resemblance to the geopackage.

Reply:
I agree on the personal geodatabase resemblance, will mention this. Thanks!

As a sidenote, the [personal geodatabase](https://desktop.arcgis.com/en/arcmap/10.3/manage-data/geodatabases/a-comparison-of-geodatabase-types.htm) has its limitations compared to GPKG; most obvious are: windows-only, closed format, limited to 250-500 MB (GeoPackage [can](http://www.geopackage.org/spec/#_sqlite_container) handle 140 TB if the filesystem can).

Another related sidenote. Found this in the FAQ on geopackage.org:

> Q. Does GeoPackage replace Shapefile?

> A. It could but it doesn’t have to. If all you need is simple exchange and display then GeoPackage may be overkill and something like GeoJSON may be more appropriate. If you are looking for database capabilities like random access and querying then GeoPackage is a platform-independent, vendor-independent choice. GeoPackage was carefully designed this way to facilitate widespread adoption and use of a single simple file format by both commercial and open-source software applications — on enterprise production platforms as well as mobile hand-held devices.

So: they do stress the particularities of each format. In applications, books and blogs, you can observe that 'shapefile by default' is (mostly?) replaced by 'GeoPackage as default'.

Reply:
Handled by aa575f0